specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json

```
@@ -0,0 +1,1796 @@
+            "$ref": "#/parameters/sentShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "sentShare",
+            "description": "The sent share to create.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SentShare"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SentShare"
+            },
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SentShares_Create": {
+            "$ref": "./examples/SentShares_Create.json"
+          },
+          "SentShares_CreateAdlsGen2Account": {
+            "$ref": "./examples/SentShares_CreateAdlsGen2Account.json"
+          },
+          "SentShares_CreateBlobAccount": {
+            "$ref": "./examples/SentShares_CreateBlobAccount.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "SentShare"
+        ],
+        "summary": "Deletes a sent share.",
+        "description": "Delete a sent share",
+        "operationId": "SentShares_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/sentShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SentShares_Delete": {
+            "$ref": "./examples/SentShares_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/sentShares/{sentShareId}/sentShareInvitations": {
+      "get": {
+        "tags": [
+          "SentShareInvitation"
+        ],
+        "summary": "List all sent share invitations in a sent share",
+        "description": "List sent share recipients",
+        "operationId": "SentShares_GetAllInvitations",
```

The verb in the operationId for `get` on a collection endpoint should be "list".
```suggestion
        "operationId": "SentShares_ListInvitations",
```